WebApiBackend/WebApiBackend/WebApiBackend.csproj

```diff
@@ -18,18 +18,26 @@
     <Compile Remove="Migrations\20200310044947_add-migration ManyToManyMigration.Designer.cs" />
     <Compile Remove="Migrations\20200310045018_ManyToManyMigration.cs" />
     <Compile Remove="Migrations\20200310045018_ManyToManyMigration.Designer.cs" />
+    <PackageReference Include="AutoMapper" Version="9.0.0" />
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.2" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.2" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="3.1.2" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.0.0" />
```

Comment:
Probably should remove this since we're using sqlite

Reply:
Removed